p384/tests/projective.rs

```diff
@@ -0,0 +1,142 @@
+//! Projective arithmetic tests.
+
+#![cfg(all(feature = "arithmetic", feature = "test-vectors"))]
+
+use elliptic_curve::{
+    sec1::{self, ToEncodedPoint},
+    PrimeField,
+};
+use p384::{
+    test_vectors::group::{ADD_TEST_VECTORS, MUL_TEST_VECTORS},
+    AffinePoint, ProjectivePoint, Scalar,
+};
+
+/// Assert that the provided projective point matches the given test vector.
+// TODO(tarcieri): use coordinate APIs. See zkcrypto/group#30
+macro_rules! assert_point_eq {
+    ($actual:expr, $expected:expr) => {
+        let (expected_x, expected_y) = $expected;
+
+        let point = $actual.to_affine().to_encoded_point(false);
+        let (actual_x, actual_y) = match point.coordinates() {
+            sec1::Coordinates::Uncompressed { x, y } => (x, y),
+            _ => unreachable!(),
+        };
+
+        assert_eq!(&expected_x, actual_x.as_slice());
+        assert_eq!(&expected_y, actual_y.as_slice());
+    };
+}
```

Comment:
@str4d workaround I ended up with for comparing curve points to test vectors.

I suppose I can just change the structure of the test vectors to include SEC1-encoded points.